### DISCUSSION TITLE
Game crashes when slider velocity is adjusted in the editor

### DISCUSSION THREAD

**Discussion by dicloflom** [ppy/osu#24699] [Q&A] [answered]
While using the editor, after the last update, when i try to change the velocity of the slider in newly created beatmaps(in the new laser version) the game gives a warning. If i double click, it crashes.

Sorry if my english is bad. I will give the logs, videos of the event and the beatmap files for you to understand or recreate the issue.
_videos_
https://youtu.be/48uxuzo4L5s
https://www.youtube.com/watch?v=Vxaw-p7LS4Y

_logs_
[performance.log](https://github.com/ppy/osu/files/12500163/performance.log)
[runtime.log](https://github.com/ppy/osu/files/12500164/runtime.log)
[updater.log](https://github.com/ppy/osu/files/12500165/updater.log)
[input.log](https://github.com/ppy/osu/files/12500166/input.log)
[network.log](https://github.com/ppy/osu/files/12500167/network.log)

_map exports, both olz and osz files for each map_
[exports.zip](https://github.com/ppy/osu/files/12500191/exports.zip)

**Comment by bdach** [comment, marked as answer]
https://github.com/ppy/osu/issues/24700
